src/e2e/e2e.test.js

```diff
@@ -0,0 +1,229 @@
+      executablePath:
+        "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
+      headless: false,
+      args: ["--start-fullscreen"],
+    });
+    page = await browser.newPage();
+    await page.goto(`${CLIENT_URL}/login`);
+  });
+
+  afterAll(() => {
+    browser.close();
+  });
+
+  describe("Login", () => {
+    beforeEach(async () => {
+      await page.goto(`${CLIENT_URL}/login`);
+    });
+
+    afterEach(async () => {
+      await page.evaluate((email) => {
+        fetch("/api/users/delete", {
+          method: "DELETE",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({ email }),
+        });
+      }, mockUser.email);
+    });
+
+    it("should register a new user", async () => {
+      await page.waitForSelector("[data-cy=register-button]");
+      const registerButton = await page.$("[data-cy=register-button]");
+      await registerButton.click();
+
+      await page.type("[data-cy=register-name]", mockUser.name);
+      await page.type("[data-cy=register-email]", mockUser.email);
+      await page.type("[data-cy=register-password]", mockUser.password);
+      await page.click("[data-cy=submit-button]");
+      await page.waitForTimeout(500);
+      const message = await page.$eval(
+        "[data-cy=message]",
+        (el) => el.textContent,
+      );
+      expect(message).toContain("유저가 등록 되었습니다.");
+    });
+
+    it("should login a user", async () => {
+      await page.waitForSelector("[data-cy=login-button]");
+      const loginButton = await page.$("[data-cy=login-button]");
+      await loginButton.click();
+
+      await page.click("[data-cy=login-button]");
+      await page.type("[data-cy=login-email]", mockUser.email);
+      await page.type("[data-cy=login-password]", mockUser.password);
+      await page.click("[data-cy=submit-button]");
+
+      await Promise.all([
+        page.waitForNavigation(),
+        page.click("[data-cy=submit-button]"),
+      ]);
+
+      await page.waitForTimeout(500);
+      await page.screenshot({ path: "screenshot.png" });
+
+      const currentUrl = await page.url();
+      expect(currentUrl).toEqual(`${CLIENT_URL}/`);
+    });
+  });
+
+  describe("RoomMaker", () => {
+    let browser;
+    let page;
+    const mockUser = {
+      name: "Test man",
+      email: "testman@example.com",
+      password: "password123",
+    };
+
+    const login = async () => {
+      await page.waitForSelector("[data-cy=login-button]");
+      await page.click("[data-cy=login-button]");
+      await page.type("[data-cy=login-email]", mockUser.email);
+      await page.type("[data-cy=login-password]", mockUser.password);
+      await page.click("[data-cy=submit-button]");
+      await page.waitForNavigation();
+    };
+
+    beforeEach(async () => {
+      browser = await puppeteer.launch({
+        executablePath:
+          "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
```

Comment:
```suggestion
          CHROME_EXECUTABLE_PATH
```
구글 크롬 실행 파일의 경로를 상수화 해주는 것이 좋을것 같습니다!